src/getCoinCombination.test.js

```diff
@@ -7,4 +7,67 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('cents = 0', () => {
```

Comment:
the summary of the test is not clear, rewatch the videos about naming tests:
https://mate.academy/en/learn/javascript-testing/tdd-basics-js?section=video&videoId=1764
https://mate.academy/en/learn/javascript-testing/tdd-basics-js?section=video&videoId=1765